jammy/debian/changelog

```diff
@@ -1,71 +0,0 @@
-sdformat13 (13.5.0-1~jammy) jammy; urgency=medium
```

Comment:
I think there needs to be at least one entry in a changelog for our deb-builders to work:

* https://github.com/gazebo-release/gz-sensors8-release/blob/main/jammy/debian/changelog

Reply:
Fixed in https://github.com/gazebo-release/sdformat14-release/pull/1/commits/fdb76cb117183737c9d111e3d0935cf1f65eb669